Tech Lead/Minutes/20241203.md

```diff
@@ -0,0 +1,39 @@
+# Technical Lead Development Team call 3 December 2024
+
+## Participants
+
+- Attendees: @schlunma @bouweandela @rbeucher @valeriupredoi @charles-turner-1 @flicj191 @ehogan @sloosvel
+- Absentees: none
+- Chair: @valeriupredoi
+- Minutes taker: @valeriupredoi
+
+## Action Items
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+| 2024-12-03 | @rbeucher | Arrange meeting on Intake | Pending |
+| 2024-12-03 | @ehogan   | Split docs meeting | Pending |
+| 2024-08-13 | all       | Review [RTW pull request](https://github.com/ESMValGroup/ESMValTool/pull/3210)|Done, thank you everyone! :tada: |
+
+## Agenda and topics discussed
+
+- Review of Pending Action Items
+- @ehogan: From last time: A new Geoscience RSE community has recently been formed (https://society-rse.org/weather-and-climate-rsecon24-birds-of-a-feather-session-retrospective-blog-post/) and we are interested in organising a townhall meeting for RSEs at EGU25 next year; will there be an ESMValTool presence there and should we get involved?
+- @ehogan: I have created a RTW project board (https://github.com/orgs/ESMValGroup/projects/27/views/1) to help @sloosvel and I (others are welcome!) work together to get RTW working for v2.12.0 release :tada: @bettina-gier: where to put smaller test recipes, Emma says tag her - still to be decided;
+@bouweandela: test ones on CircleCI - but data must be small, since they get downloaded/run every night, see [CI test configuration](https://github.com/ESMValGroup/ESMValTool/blob/ba9cfe426f9f2a97dba672784ccba8462eea7d6e/.circleci/config.yml#L177-L231) -> these test Core mostly;
+- @ehogan: Chris is back working on ESMValTool until he finishes with his Apprenticeship (~May 2025). Is there anything, perhaps something non-critical-path-Dask-related, that he could get involved in? - @bouweandela: could run all recipes with the `dask.Distributed` scheduler, not all work, it's a case-by-case challenging work, @ehogan proposes using the simple scheduler first; @bouweandela also proposes `pyproject.toml`-ing ESMValTool; @schunma is very keen on Dask optimization with `Distributed`, @bouweandela proposes Manu work mostly on it, with Chris testing;
+- @ehogan: Using the new Software Carpentry template [ESMValTool Tutorial Issue #335](https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/335)
+- @ehogan: Should we arrange a meeting to come up with a documentation design following this [comment on "Agenda for ESMValTool community workshop: 19-21 November"](https://github.com/ESMValGroup/Community/discussions/192#discussioncomment-11258741) - @rbeucher is keen to get Felicity onboard; Emma will organize the first meeting on this via a Doodle, this will be a **to-do item** here;
```

Comment:
```suggestion
- @ehogan: Should we arrange a meeting to come up with a documentation design following this [comment on "Agenda for ESMValTool community workshop: 19-21 November"](https://github.com/ESMValGroup/Community/discussions/192#discussioncomment-11258741) - @rbeucher is keen to get Felicity onboard; Emma will organize the first meeting on this via a Doodle, this will be an **action item** here;
```